test/unittests/test_corosync.py

```diff
@@ -148,26 +148,68 @@ def test_find_configured_ip_exception(self, mock_open_file, mock_conf, mock_pars
         mock_ip_local.assert_called_once_with(False)
         mock_search.assert_called_once_with("nodelist.node.ring[0-9]*_addr", "nodelist.node.ring0_addr")
 
-    def test_add_node_ucast(self):
-        from crmsh.corosync import add_node_ucast, get_values
+    @mock.patch('crmsh.corosync.find_configured_ip')
+    def test_add_node_ucast_exception(self, mock_find_ip):
```

Comment:
I wrote a comment here before, but it looks like I didn't save it...
So...
I think we don't need this unit test. It doesn't test anything relevant. It checks that the exception is re-raised. This will happen always, with any exception, so it's not a real test.
We should only test this, if the `IPAlreadyConfiguredError` was caught using `try` and re-raised later (or any other operation).
Basically, I would just remove this test case

Reply:
I suggest leave this test, since at least this test increase the test coverage:)

Reply:
Really? How is that possible? This test code branch doesn't run any different code compared with `test_add_node_ucast`